src/Entity/AlbumFile.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+namespace App\Entity;
+
+use App\Repository\AlbumFileRepository;
+use Doctrine\Common\Collections\ArrayCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+#[ORM\Entity(repositoryClass: AlbumFileRepository::class)]
+class AlbumFile
+{
+    #[ORM\Id]
+    #[ORM\GeneratedValue]
+    #[ORM\Column(type: 'integer')]
+    private $id;
+
+    #[ORM\Column(type: 'datetime_immutable')]
+    private $created_at;
```

Comment:
Same `datetime_immutable` to `datetime`
Les variables en PHP sont toujours en pascalCase.